onnxruntime/test/providers/xnnpack/xnnpack_basic_test.cc

```diff
@@ -182,22 +185,28 @@ static void RunModelTest(
                             helper.feeds_, params);
 }
 
-static void RunModelTestWithPath(const ORTCHAR_T* ort_model_path, const char* graph_name, float scale_factor = 1.0f) {
-  std::function<void(const Graph&)> verify = [](const Graph& graph) -> void {
-    ASSERT_EQ(graph.NumberOfNodes(), 5) << "Transpose*2 + dq +q +qlinearconv "
-                                           "leaving 5 nodes.";
-  };
+static void RunModelTestWithPath(const ORTCHAR_T* ort_model_path, const char* graph_name,
+                                 std::function<void(const Graph&)> graph_verifier = nullptr,
+                                 float err_tolerance = .2f) {
   EPVerificationParams params;
```

Comment:
nit: call it abs_err_tolerance given floating point comparisons usually involve absolute and relative tolerances.